### ISSUE TITLE
Overrides Sublime Autocomplete

### ISSUE THREAD

**Issue by tinversenorm** [tushortz/JavaIME#6]
This package overrides sublime's default autocomplete. So when I type a variable name, Sublime no longer autocompletes it. Can this be fixed?

**Comment by rsyqvthv** [comment]
yes, same here, would like to remain the native autocomplete for variable matches. thanks.

**Comment by tushortz** [comment]
@tinversenorm @2anyone can you show me an example of what overrides what?

**Comment by tinversenorm** [comment]
I've uninstalled the plug in because of it, but in essence, when I start typing a variable name, Sublime would only suggest the Java IME autocomplete options but not my previous variable names.

**Comment by tinversenorm** [comment]
Ideally, it would be more like IntelliJ, which suggests both Java classes and variable names.
